force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -351,18 +456,28 @@ public with sharing class FlowControlLoopsCollections {
 
         // Add key-value pairs to the Map
         // add John Doe => 50000
+        employeeSalaries.put('John Doe', 50000);
         // add Jane Smith => 60000
+        employeeSalaries.put('Jane Smith', 60000);
         // add Sam Brown => 55000
+        employeeSalaries.put('Sam Brown', 55000);
         // add Alice Johnson => 65000
+        employeeSalaries.put('Alice Johnson', 65000);
```

Comment:
Since the values are the same try creating a variable that can be used on both question 19 and 20 
This will require a top level static variable and is more advanced than things we talked about.